src/__stories__/Index.stories.mdx

```diff
@@ -1,7 +1,46 @@
-import { Description, Meta } from '@storybook/addon-docs/blocks'
+import { Meta } from '@storybook/addon-docs/blocks'
 
 <Meta title="Home" />
 
 # Welcome to Scaleway UI
 
-<Description />
+Scaleway UI is a set of React components you can use to build your application. It is used to build our [Scaleway Console](https://console.scaleway.com).
+
+## Getting started
+
+### Installation
+
+With npm:
+
+```bash
+npm install @scaleway/ui
+```
+
+With yarn:
```

Comment:
Really a small detail but maybe we should put yarn first?

Reply:
So according to the team, we drop npm documentation and use only yarn